IOHexperimenter/Extension.md

```diff
@@ -5,8 +5,8 @@ parent: IOHexperimenter
 permalink: /IOHexp/extension/
 --- 
 
-
-## <a name="adding-new-problems"></a>Add new problems
+<a name="adding-new-problems"></a>
+## Adding new problems
```

Comment:
Put a reference to this in the section about using problems as well, so it is clear that it is possible.